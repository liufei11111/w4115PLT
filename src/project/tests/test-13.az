String s;
s = " ";
s = "";
s = "asdf";
s = "!@asf#$%^as325df34&*()1sfd";
s = "as\\nasd";
s = "asd\r";
