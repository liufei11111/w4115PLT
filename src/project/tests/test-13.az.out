
public static int main(int argc, String argv)
{
try{String s=null;
s = " ";
s = "";
s = "asdf";
s = "!@asf#$%^as325df34&*()1sfd";
s = "as\\nasd";
s = "asd\r";
}catch(Exception e){return -1;}}

